Website/src/rendering/src/stories/Sponsors/SponsorInformation.stories.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { SESSIONS } from '../../models/mock-sessions';
```

Comment:
As the mock sessions are now only used in stories, it would be good to move them from the "models" folder to the root of the "stories" folder.